### DISCUSSION TITLE
Custom tests in Trivy Operator (host filesystem)

### DISCUSSION THREAD

**Discussion by xzizka** [aquasecurity/trivy-operator#1776] [Help & Support] [answered]
Hello,

I would like to add custom tests to Trivy Operator. We want to use it as a complex tool, that will be doing a whole bunch of tests we need for the Kubernetes platform. It is possible to cover Kubernetes tests with the CIS benchmark, but we are looking for one specific test.
We need to check specific files and their access permissions. E.g. we need to check the access permissions and ownership of the file `/etc/kubernetes/admin.conf` and some other custom config files. 
This file is located on a host filesystem and to check its permissions means mount its location to a pod and then perform the test.
This is possible to achieve with a custom plugin when trivy is called via CLI, but I didn't find any way how to achieve something like this trivy-operator. Does there exist any way how to achieve that?

Thank you

**Comment by chen-keinan** [comment, marked as answer]
@xzizka please checkout here the [compliance specs](https://github.com/aquasecurity/trivy-operator/tree/main/deploy/helm/templates/specs) we support

**Reply by xzizka** [reply]
Thank you
